hr_timesheet_no_closed_project_task/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Timesheet No Closed Project-Task",
+    'summary': """
+        Prevent to select closed project or task on timesheet line""",
+    "version": "8.0.1.0.0",
+    "author": "ACSONE SA/NV,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.acsone.eu",
+    "category": "Human Resources",
+    "license": "AGPL-3",
+    "depends": [
+        "hr_timesheet_task",
+        "hr_timesheet_invoice",
+        "account_analytic_project",
+        "project_stage_closed",
```

Comment:
The `project_stage_closed` module is [built to work](https://github.com/OCA/project/pull/142/files#diff-9899655bb6670bd911675a6ef4adcb7cR27) only if `sale_service` is installed.
Forcing this dependency here is inappropriate.
In Odoo standard closed Tasks are Tasks in a "folded" stage. I think you should stick to that.